src/main/java/com/atlassian/jira/cloud/jenkins/config/ConfigManagementLink.java

```diff
@@ -205,7 +238,20 @@ public void doSaveConfiguration(final StaplerRequest req, final StaplerResponse
             doIndex(req, res);
             return;
         }
-        res.sendRedirect("/jenkins/manage/");
+        redirectToManagePage(req, res);
+    }
+
+    public void redirectToManagePage(final StaplerRequest req, final StaplerResponse res) {
+        String jenkinsRootUrl = Jenkins.get().getRootUrl();
```

Comment:
WE need to muck around a bit here because different servers have different paths for where there manage screen is :sigh: but this catchs all